gym_ai2thor/envs/ai2thor_env.py

```diff
@@ -156,8 +167,22 @@ def step(self, action, verbose=True):
                     inventory_changed_str = ''
                 print('{}: {}. {}'.format(
                     action_str, interaction_obj['objectType'], inventory_changed_str))
+        elif 'Rotate' in action_str:
```

Comment:
why not "startswith"? I don't know if it makes a difference in speed, but it certainly is the way it is implemented

Reply:
ok I did it, but I also changed all the ___Object ones as well if we want to be consistent.
Since `action_str.endswith('Object')`
we can change the 4 interaction if statements with object actions to:
`action_str.startswith('Put')`

Reply:
I like it, please do